egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -564,7 +565,7 @@ def get_model_combine_iters(num_iters, num_epochs,
         in the final model-averaging phase.  (note: it's a weighted average
         where the weights are worked out from a subset of training data.)"""
 
-    approx_iters_per_epoch_final = num_archives/num_jobs_final
+    approx_iters_per_epoch_final = float(num_archives) / num_jobs_final
```

Comment:
Isn't adding `float` here redundant after `from __future__ import division`?

Reply:
Thanks for pointing out. I will resolve this once @vimalmanohar suggests which one to retain.